Frends.CSV.Create/Frends.CSV.Create/Create.cs

```diff
@@ -90,11 +90,116 @@ private static string ListToCsvString(List<List<object>> inputData, List<string>
         return csvString.ToString();
     }
 
-
     private static string JsonToCsvString(string json, CsvConfiguration config, Options options, CancellationToken cancellationToken)
     {
-        List<Dictionary<string, string>> data = JsonConvert.DeserializeObject<List<Dictionary<string, string>>>(json);
+        //stringify numbers
+        var jsonElement = System.Text.Json.JsonSerializer.Deserialize<JsonElement>(
+            json,
+            new JsonSerializerOptions { AllowTrailingCommas = true, }
+        );
+        string stringifiedJson = System.Text.Json.JsonSerializer.Serialize(
+            jsonElement,
+            new JsonSerializerOptions { Converters = { new NumberToStringConverter() } }
+        );
+
+        JToken input = JsonConvert.DeserializeObject<JToken>(
+            stringifiedJson,
+            new JsonSerializerSettings { DateParseHandling = DateParseHandling.None, }
+        );
+        input = input is JObject ? new JArray() { input } : input;
+
+        //compute what are max sizes of arrays in all objects
+        var limits = new Dictionary<string, int>();
+        foreach (var row in input)
```

Comment:
The limits calculation can be moved to its own method. This method is already quite big.